katello/katello-change-hostname

```diff
@@ -170,6 +171,12 @@ def precheck
     fail_with_message("There is a problem with the server, please check 'hammer ping'")
   end
 
+  STDOUT.puts "\nChecking credentials"
```

Comment:
I was trying to think if there is a more generic command we can use, but can't really find one! Maybe @akofink can think of one (we are just trying to check if the username/password credentials are correct)

Reply:
`hammer ping` ?

Reply:
`hammer ping` does not require credentials, and it does not error out for wrong ones either.
we could do `hammer user info --name #{@options[:username]}`, but that feels equally ugly.

Reply:
I'm fine with `capsule list`, since we will run that later.